src/features/defi/providers/foxy/components/FoxyManager/Overview/FoxyOverview.tsx

```diff
@@ -78,7 +78,7 @@ export const FoxyOverview: React.FC<FoxyOverviewProps> = ({
   })
   const rewardAsset = useAppSelector(state => selectAssetById(state, rewardAssetId))
   const marketData = useAppSelector(state => selectMarketDataById(state, stakingAssetId))
-  const cryptoAmountAvailable = bnOrZero(foxyBalance).div(`1e${stakingAsset.precision}`)
+  const cryptoAmountAvailable = bnOrZero(foxyBalance).div(bn(10).pow(stakingAsset?.precision ?? 0))
```

Comment:
Not a big deal since precision amounts are going away but might as well
```suggestion
  const cryptoAmountAvailablePrecision = bnOrZero(foxyBalance).div(bn(10).pow(stakingAsset?.precision ?? 0))
```